README.md

```diff
@@ -26,7 +26,7 @@ The current compatibility matrix reads as follows:
 
 | `lndclient` git tag          | `lnd` version in `go.mod` | minimum required `lnd` version | 
 | ---------------------------- | ------------------------- | ------------------------------ |
-| [`v0.14.2-0`](https://github.com/lightninglabs/lndclient/blob/v0.14.2-0) | `lnd @ 4a62b7cf8c69` | `v0.14.0-beta` |
+| [`v0.14.2-1`](https://github.com/lightninglabs/lndclient/blob/v0.14.2-1) | `v0.14.2-beta` | `lnd @ 99378b22c3a3` |
```

Comment:
nit: can remove/replace the `lnd@...` string and just use `v0.14.2-beta`?

Reply:
Done.